src/wscearth/views.py

```diff
@@ -6,6 +6,7 @@
 import flask_cachecontrol
 import flask_googlemaps
 from influxdb_client_3 import InfluxDBClient3
+import simplejson as json
```

Comment:
What's the advantage of `simplejson` over python's inbuilt `json`?

Reply:
For the most part they're almost identical. At some point, early Python 2.x series, it was included into the stdlib. However, it hasn't kept up like simplejson on speed or correctness.

For us, pandas loves converting to `NaN` but that' not valid JSON (breaks the JS parser) and using `builtin.ignore_nan` will instead raise ValueError whereas simplejson makes peace and converts to a null.